gratipay/billing/exchanges.py

```diff
@@ -198,12 +198,20 @@ def get_ready_payout_routes_by_network(db, network):
                  ) > 0
 
 
-                OR -- Include green-lit Gratipay 1.0 balances
+                OR -- Include team members
+
+                (SELECT count(*)
+                   FROM current_takes ct
```

Comment:
What if... I had a take setup before payday started (which is what payday will consider throughout), and then I leave the team before masspay happens. I won't be included in this list, and hence won't receive a payout at all...